docker/Dockerfile

```diff
@@ -22,11 +22,16 @@ RUN pacman -Syyuu --noconfirm --needed --overwrite \*
 # Add a script that installs packages
 COPY usr/local/bin/install-package /usr/local/bin/install-package
 
+# Install latest archlinux-keyring package so we have the most up-to-date keys
+RUN install-package wget
+RUN wget https://archive.archlinux.org/repos/last/core/os/x86_64/archlinux-keyring-20201028-1-any.pkg.tar.zst \
```

Comment:
Is this URL something we'll have to update regularly? Or is there something like a /latest.pkg.tar.zst we could use?

Reply:
We might be able to use 
```
RUN wget https://www.archlinux.org/packages/core/any/archlinux-keyring/download/ -O /var/tmp/archlinux-keyring.pkg.tar.zst
RUN pacman -U /var/tmp/archlinux-keyring.pkg.tar.zst --noconfirm
```

Haven't tested this though

Reply:
I added it and tested locally and it works

Reply:
Just tested this. was able to build on my machine without issue